Kuroba/app/src/main/java/com/github/adamantcheese/chan/ui/controller/ImageViewerController.java

```diff
@@ -125,7 +127,9 @@ public ImageViewerController(Loadable loadable, Context context, Toolbar toolbar
         this.toolbar = toolbar;
         this.loadable = loadable;
 
-        presenter = new ImageViewerPresenter(this);
+        AudioManager audioManager = (AudioManager) context.getSystemService(AUDIO_SERVICE);
```

Comment:
You can move all this code up into the ImageViewerPresenter constructor itself. No need to modify extra files if it's only used in one location.

Reply:
How do I get the context in the ImageViewerPresenter?

Reply:
You don't need the exact context for system level managers, so using `AndroidUtils.getAppContext()` should work fine.

Reply:
> You don't need the exact context for system level managers, so using `AndroidUtils.getAppContext()` should work fine.

Alright, done. Thanks for the tip!